js/src/ui/Modal/modal.js

```diff
@@ -49,6 +50,14 @@ class Modal extends Component {
     waiting: PropTypes.array
   }
 
+  componentDidMount () {
+    const e = ReactDOM.findDOMNode(this.refs.dialog);
```

Comment:
Would have preferred `const element = ReactDOM...` instead of the single `e` that could mean error, event, element, etc. (Not really gaining anything by being cryptic)